### ISSUE TITLE
Add options to repair and uninstall

### ISSUE THREAD

**Issue by doggybootsy** [BetterDiscord/Installer#202]
**Is your feature request related to a problem? Please describe.**
kinda, like if a plugin/theme is having issues the installer can have a option in repair that would disable all plugins and themes

**Describe the solution you'd like**
Add options to repair and uninstall
Like repair can have the options of `disable addons and reinstall bd` (addons being themes and plugins) and `reinstall bd` and uninstall can have a way to fully remove bd and just remove it off of discord

**Describe alternatives you've considered**
Basically something like those

**Additional context**
no

**Comment by dav1312** [comment]
Looks similar to #189

**Comment by doggybootsy** [comment]
It does about the repair part

**Comment by zerebos** [comment]
> and uninstall can have a way to fully remove bd and just remove it off of discord

Can you elaborate on this?

**Comment by doggybootsy** [comment]
> fully remove bd

Deletes the bd folder in appdata (or os specific spot)

> just remove it off of discord

Like how uninstalling is right now

**Comment by dav1312** [comment]
> > just remove it off of discord
> 
> Like how uninstalling is right now

Maybe just call it "Disable"

**Comment by doggybootsy** [comment]
Ya
